_sub/compute/eks-addons/main.tf

```diff
@@ -144,3 +194,31 @@ resource "kubernetes_annotations" "gp2-not-default" {
     aws_eks_addon.aws-ebs-csi-driver
   ]
 }
+
+resource "kubernetes_storage_class" "csi-efs" {
+  metadata {
+    name = "csi-efs"
+    annotations = {
+      "storageclass.kubernetes.io/is-default-class" = "false"
+    }
+  }
+  storage_provisioner    = "efs.csi.aws.com"
+  reclaim_policy         = "Delete" # ?
```

Comment:
I think the default should be Retain.

Reply:
@avnes why should this be different from what we offer through the EBS as we do here https://github.com/dfds/infrastructure-modules/blob/096983220481d55f2c15ee278e6f21ad46d9be80/_sub/compute/eks-addons/main.tf#L119?

Reply:
OK, if it is the same, then in is OK